doc/page-cmakefunctions.h

```diff
@@ -0,0 +1,126 @@
+/**
+
+\page CMake-Functions FaceLift CMake API Reference
+
+\section public-functions Functions for FaceLift Users
+
+The following functions are public and are meant to be used by FaceLift users.
+
+\code
+facelift_add_interface(<TargetName>
+                       INTERFACE_DEFINITION_FOLDER dir
+                       [LINK_LIBRARIES lib1 [lib2...]]
+                       [IMPORT_FOLDERS folder1 [folder2...]])
+\endcode
+
+This function creates a new facelift interface library and generates the corresponding modules (like ipc, types, devtools, etc.)
+
+\param TargetName The name of the interface library that will be created
+\param INTERFACE_DEFINITION_FOLDER The folder contains the QFace files for which some code is going to be generated
```

Comment:
The folder containing the QFace interface definition files from which the library should be generated.